gffquant/annotation/genecount_annotator.py

```
@@ -0,0 +1,62 @@
+""" module docstring """
+import logging
+
+from .count_annotator import CountAnnotator
+from .count_writer import CountWriter
```

Ruff (0.8.2)

`.count_writer.CountWriter` imported but unused

Remove unused import: `.count_writer.CountWriter`

(F401)